js/elm-package.json

```diff
@@ -11,7 +11,8 @@
         "elm-lang/animation-frame": "1.0.1 <= v < 2.0.0",
         "elm-lang/core": "5.1.1 <= v < 6.0.0",
         "elm-lang/html": "2.0.0 <= v < 3.0.0",
-        "elm-lang/websocket": "1.0.0 <= v < 2.0.0"
+        "elm-lang/websocket": "1.0.0 <= v < 2.0.0",
+        "mgold/elm-nonempty-list": "3.1.0 <= v < 4.0.0"
```

Comment:
Seems like this dependency doesn't do anything?

Reply:
Oh yes. Good catch.